alby/alby_oauth_service.go

```diff
@@ -937,11 +937,13 @@ func (svc *albyOAuthService) requestAutoChannel(ctx context.Context, url string,
 	type autoChannelRequest struct {
 		NodePubkey      string `json:"node_pubkey"`
 		AnnounceChannel bool   `json:"announce_channel"`
+		NodeType        string `json:"node_type"`
 	}
 
 	newAutoChannelRequest := autoChannelRequest{
 		NodePubkey:      pubkey,
 		AnnounceChannel: isPublic,
+		NodeType:        svc.cfg.GetEnv().LNBackendType,
```

Comment:
shouldn't we use those getters on the cfg for this? 

wondering because of that issue where the DB should be prioritized aver the env vars - which means we should always use the data from the DB I guess?

Reply:
yes, sorry!